packages/react/rollup.config.js

```diff
@@ -13,7 +13,6 @@ export default {
   output: {
     dir: 'lib',
     format: 'cjs',
-    preserveModules: true,
```

Comment:
@scurker this flag causes the build to break. For some strange reason, it emits `lib/src` and `lib/node_modules/highlight.js`. I don't know why we added it in the first place. Is it safe to remove?

Reply:
The old webpack code preserved the directory structure so I was trying to keep the same structure when migrating from webpack, so this option kept that output the same.

The only downside to not having it is debugging issues with cauldron, you'll see something like `cauldron/lib/index.js:23484` instead of `cauldron/lib/components/foo.js:234`. I'm not sure why it's just now causing issues but we can probably remove it.

Reply:
It's causing issues now because of how we're "registering" languages in the Code component. I don't fully understand it either 🤷